base/build.gradle

```diff
@@ -61,6 +61,8 @@ dependencies {
 
     api 'androidx.room:room-runtime:2.2.2'
     api 'androidx.room:room-testing:2.2.2'
```

Comment:
Use `testApi` or `androidTestApi` - depends in which tests it will be used: unit or android.
You can also extract version to `def` - look below.

Reply:
Not changed :(

Reply:
I didn't add this dependencies. I still need change it?

Reply:
Indeed, you don't have to do it, but it is good approach.

Reply:
Another time :)